### PR TITLE
feat(PPDSC-2396): update no pointer distance for the tooltip and popover components

### PR DESCRIPTION
**Pull request by jannuk59**
[PPDSC-2326](https://nidigitalsolutions.jira.com/browse/PPDSC-2396)

- Updated the condition to `...(distance ? [offset(distance)] : []),` to set the distance when there is noPointer.
- By default the distance between the Button and table will be `'space020'`

**TO DO**
Check whether we can set the customisable distance when we have noPointer for the `Tooltip` and `Popover` components

**I have done:**
- [ ] Written unit tests against changes
- [x] Written functional tests against the component and/or NewsKit site
- [x] Updated relevant documentation

**I have tested manually:**
- [x] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [x] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2396-update-pointer-distance
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2396-update-pointer-distance/storybook

File: src/popover/__tests__/popover.stories.tsx [hunk 1]
```diff
@@ -231,10 +231,32 @@ export const StoryPopoverDefault = () => (
...
+            overrides={{distance: 'space080', maxWidth: '300px'}}
+          />
+        </div>
+        <div>
+          <StorybookSubHeading>
+            Popover - no pointer distance (zero)
+          </StorybookSubHeading>
+          <BoundedPopover
+            hidePointer
+            overrides={{distance: 'space0', maxWidth: '300px'}}
```
Comment (evgenitsn) [inline src/popover/__tests__/popover.stories.tsx:256]: `space0` is invalid token, can we update this to `space000`?

File: src/tooltip/__tests__/tooltip.stories.tsx [hunk 2]
```diff
@@ -618,12 +618,38 @@ StoryTooltipPlacementsVisualTest.storyName = 'tooltip-placements-visual-test';
...
+      </Button>
+    </Tooltip>
+    <StorybookSubHeading>
+      Tooltip - no pointer distance (zero)
+    </StorybookSubHeading>
+    <Tooltip
+      content="Tooltip content"
+      placement="right"
+      hidePointer
+      overrides={{distance: 'space0'}}
```
Comment (evgenitsn) [inline src/tooltip/__tests__/tooltip.stories.tsx:647]: same here

**Review by evgenitsn — CHANGES_REQUESTED** [review]

**Comment by evgenitsn** [comment]
Can you please update the PR name to follow the convention?
Example: https://github.com/newscorp-ghfb/newskit/pull/370

**Review by mutebg — APPROVED** [review]

**Review by evgenitsn — APPROVED** [review]

**Pull request merged**